pkg/integration/tests/interactive_rebase/edit_non_todo_commit_during_rebase.go

```diff
@@ -0,0 +1,37 @@
+package interactive_rebase
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var EditNonTodoCommitDuringRebase = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Tries to edit a non-todo commit while already rebasing, resulting in an error message",
```

Comment:
I only added a test for "edit" as an example; we could add tests for all other operations (reword, squash, fixup, move up/down), but it seemed a bit too much.